assets/tests/components/propertiesPanel/vehiclePropertiesPanel.test.tsx

```diff
@@ -134,6 +134,13 @@ describe("VehiclePropertiesPanel", () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test("Includes invalid bus banner when vehicle is off course", () => {
+    render(
+      <VehiclePropertiesPanel selectedVehicle={invalidVehicleFactory.build()} />
+    )
+    expect(screen.getByText("Invalid Bus")).toBeInTheDocument()
```

Comment:
suggestion(non-blocking): I'd like it if we could change this to be more semantic and use a `getByRole` to guide adjusting the content if it's possible.

```suggestion
    expect(screen.getByRole("heading", { name: "Invalid Bus" })).toBeVisible()
```

I could be wrong about "heading" being the correct role here, but I feel like there's some role that should represent this part of the card?

Reply:
Thanks for the suggestion! I added as `<h3>` since there is already a `<h2>` as the VPP header. This required some h3 styling overrides for "Invalid Bus" & block waiver card titles, open to suggestions for a more consistent way to do that! I was thinking of making the card title a `<h3>` always, but I think the level would need to depend on where the card is.